pygmt/datasets/samples.py

```diff
@@ -25,7 +25,6 @@ def list_sample_data():
     names = {
         "bathymetry": "Table of ship bathymetric observations off Baja California",
         "earth_relief_holes": "Regional 20 arc-minutes Earth relief grid with holes",
-        "fractures": "Table of hypothetical fracture lengths and azimuths",
```

Comment:
I don't think this should be removed; we still want it in the names listed with `list_sample_data()`.